en/appendices/5-0-migration-guide.rst

```diff
@@ -20,6 +20,8 @@ Deprecations
 The following is a list of deprecated methods, properties and behaviors. These
 features will continue to function in 5.x and will be removed in 6.0.
 
+n/a
```

Comment:
?

Reply:
The list is Actually empty
This clarifies it until we add something